### PR TITLE
[CI]Label stale issues and PRs

### PR DESCRIPTION
**Pull request by yaalsn**
### Motivation

This CI workflow can help us to filter the issue and PRs which have `Stale`  tag

### Modifications

Add a new CI workflow file called `ci-stable-issue-pr.yaml`

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation
  
- [x] `no-need-doc`

### CODE REVIEW

**Comment by codelipenghui** [comment]
Cool, thanks @yaalsn

Could you please also help put an update in https://lists.apache.org/thread/0lm9tyjqtgtvwkfowkfhbxy24nh8tyxh?

Great news!

File: .github/workflows/ci-stable-issue-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'Message to comment on stale issues. If none provided, will not mark issues stale'
```
Comment (codelipenghui) [inline .github/workflows/ci-stable-issue-pr.yaml:13]: I think the stale message should be

"The issue had no activity for 30 days, mark with Stale label."

File: .github/workflows/ci-stable-issue-pr.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+          stale-pr-message: 'Message to comment on stale PRs. If none provided, will not mark PRs stale'
```
Comment (codelipenghui) [inline .github/workflows/ci-stable-issue-pr.yaml:14]: "The PR had no activity for 30 days, mark with Stale label."

File: .github/workflows/ci-stable-issue-pr.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+          days-before-issue-stale: 30
+          days-before-pr-stale: 45
+          days-before-pr-close: -1
+          days-before-issue-close: -1
```
Comment (codelipenghui) [inline .github/workflows/ci-stable-issue-pr.yaml:18]: Use [days-before-close](https://github.com/actions/stale#days-before-close) ?

File: .github/workflows/ci-stable-issue-pr.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'Message to comment on stale issues. If none provided, will not mark issues stale'
+          stale-pr-message: 'Message to comment on stale PRs. If none provided, will not mark PRs stale'
+          days-before-issue-stale: 30
+          days-before-pr-stale: 45
```
[the remark this answers is not in the document]
Comment (codelipenghui) [reply .github/workflows/ci-stable-issue-pr.yaml:16]: If keep consistent with 30 days, we can only use [days-before-stale](https://github.com/actions/stale#days-before-stale)

File: .github/workflows/ci-stable-issue-pr.yaml (hunk 3 above)
Comment (yaalsn) [reply .github/workflows/ci-stable-issue-pr.yaml]: fixed

**Review by codelipenghui — APPROVED** [review]

**Review by hangc0276 — APPROVED** [review]
Good job!

**Review by tomscut — APPROVED** [review]

**Pull request merged**